releaseNotes/JavaScriptFeatureManagement.md

```diff
@@ -2,6 +2,12 @@
 
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 2.2.0 - Augest 21, 2025
+
+### Enhancements
+
+* Added a `Recurrence` option to the `TimeWindowFilter` to support recurring time window. This enables scenarios where a feature flag is activated based on a recurrence pattern, such as every day after 5 PM or every Friday. See the [how to Time Window Filter document](https://learn.microsoft.com/azure/azure-app-configuration/howto-timewindow-filter) for more details.
```

Comment:
`how to Time Window Filter document` -> `Enable features on a schedule` to match the actual title